vector/src/main/java/im/vector/app/features/roomprofile/members/RoomMemberListFragment.kt

```diff
@@ -49,38 +49,44 @@ class RoomMemberListFragment @Inject constructor(
     private val viewModel: RoomMemberListViewModel by fragmentViewModel()
     private val roomProfileArgs: RoomProfileArgs by args()
 
-    override fun getLayoutResId() = R.layout.fragment_room_setting_generic
-
-    override fun getMenuRes() = R.menu.menu_room_member_list
-
-    override fun onPrepareOptionsMenu(menu: Menu) {
-        val canInvite = withState(viewModel) {
-            it.actionsPermissions.canInvite
-        }
-        menu.findItem(R.id.menu_room_member_list_add_member).isVisible = canInvite
-    }
-
-    override fun onOptionsItemSelected(item: MenuItem): Boolean {
-        when (item.itemId) {
-            R.id.menu_room_member_list_add_member -> {
-                navigator.openInviteUsersToRoom(requireContext(), roomProfileArgs.roomId)
-                return true
-            }
-        }
-        return super.onOptionsItemSelected(item)
-    }
+    override fun getLayoutResId() = R.layout.fragment_room_member_list
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         roomMemberListController.callback = this
         setupToolbar(roomSettingsToolbar)
         setupSearchView()
+        setupInviteUsersButton()
         recyclerView.configureWith(roomMemberListController, hasFixedSize = true)
+        // Hide FAB when list is scrolling
+        recyclerView.addOnScrollListener(
```

Comment:
Move that code to `setupInviteUsersButton()`?

Reply:
Done.